docs/setup/savepoints.md

```diff
@@ -129,11 +144,11 @@ $ bin/flink savepoint -d :savepointPath
 
 This disposes the savepoint stored in `:savepointPath`.
 
-Note that since savepoints always go to a file system it is possible to also manually delete the savepoint via a regular file system operation. Keep in mind though that the savepoint only stores meta data that points to the actual checkpoint data. Therefore, if you manually want to delete a savepoint, you would have to include the checkpoint files as well. Since there is currently no straight forward way to figure out how a savepoint maps to a checkpoint, it is recommended to use the savepoint tool for this as described above.
+Note that since savepoints always go to a file system it is possible to also manually delete the savepoint via a regular file system operation. Up to Flink 1.2, this way a more tedious task which was solved with this tool.
```

Comment:
Up to Flink 1.2, this was a more tedious task which was performed with the savepoint tool.